README.md

```diff
@@ -86,6 +86,43 @@ username: test
 password: test
 ```
 
+#### Content Event API via FLASK
+
+1. See (dummy) events:
+```
+curl localhost:5000/events
+```
+
+2. Add event:
+```
+$ curl -d '{"extra-data-things": "boop boop boop"}' -H "Content-Type: application/json" -X POST http://localhost:5000/events
+{
+  "mimetype": "application/json",
+  "response": {
+    "extra-data-things": "boop boop boop",
+    "id": "event24",
+    "id_hash": "02776133-d49d-49cb-bfaa-67c7f61b25a1",
```

Comment:
use the full name.. `ident_hash` That's what it's called in the code. this way we don't have the same thing called different things in many places.